packages/emotion/transform/src/lib.rs

```diff
@@ -609,16 +625,32 @@ impl<C: Comments> Fold for EmotionTransformer<C> {
                                     )));
                                 }
                                 if let Some(ExprOrSpread { expr, .. }) = callee.args.get_mut(1) {
-                                    if let Expr::Object(ObjectLit { props, .. }) = expr.as_mut() {
-                                        props.extend(object_props);
-                                    } else {
-                                        callee.args.push(
-                                            Expr::Object(ObjectLit {
+                                    match expr.as_mut() {
+                                        Expr::Object(ObjectLit { props, .. }) => {
+                                            props.extend(object_props);
+                                        }
+                                        Expr::Call(_) => {
+                                            object_props.push(PropOrSpread::Spread(
+                                                SpreadElement {
+                                                    dot3_token: DUMMY_SP,
+                                                    expr:expr.to_owned()
```

Comment:
This looks like a needless clone.

Reply:
`to_owned` copies the data

Reply:
I am sorry, i don`t know how i can fix it. I am beginner in rust. Can you help me a bit

Reply:
@kdy1 thanks!